### DISCUSSION TITLE
Minecraft server not working

### DISCUSSION THREAD

**Discussion by Ulyss1974** [AlexProgrammerDE/balena-minecraft-server#149] [Q&A]
Hi!
I've just instaled a Minecraft server on my raspberry pi 4 8Gb.  I used the balenacloud service and the one click deployment but I'm not able to play on my server.
When I try to connect on the server I get this message: "io.netty.channel.AbstractChannel$AnnotatedConnectExeption: Connection refused: no further informations: 10.0.0.36/".
Also in the logs of my balenacloud dashboard I get this message: "Error: Invalid or corrupt jarfile paper.jar".
It would probably be bether to use the manual instalation but I really don't know how to do that.  Is there any body who could help whit my issue?

I finaly managed to install evrything manualy but I get the same errors and it's still not working

**Comment by viedok** [comment]
Hey ! I've been using this software for over a year now so I know it pretty well. If you want you can contact me on discord and I'll try to help you :) (username : taediumvitae#6515)
